### PR TITLE
DSP-617 Update release process

### PR DESCRIPTION
**Pull request by kilchenmann**
Resolves second part of [dsp-617](https://dasch.myjetbrains.com/youtrack/issue/DSP-617)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,22 +45,9 @@ We built the user guidelines and developer documentation with [MkDocs](https://w
...
-  - `release-patch`                  updates version to next PATCH version e.g. from 3.0.0 to 3.0.1
-- The make command will commit and push to github (you have to fill in your GitHub username and password to log when you do not use any [github token](https://docs.github.com/en/enterprise/2.15/user/articles/creating-a-personal-access-token-for-the-command-line))
-- Update README and CHANGELOG if necessary and commit the changes (currently, the CHANGELOG has to be updated manually)
-- Create new pull request and merge into master
-- Draft new release on Github. This will build, test and publish the new package on dockerhub. Additional it creates / overrides release notes on Github. Fill in:
-  - the tag version, the release title (same name)
-  - If this is a pre-release, check the box "This is a pre-release"
+A new version will be published with each github release as it's part of Github actions' workflow. To make a new release, go to <https://github.com/dasch-swiss/dsp-app/releases> and convert the draft called "[Next release](https://github.com/dasch-swiss/dsp-app/releases/tag/next-release)" by updating:
```
Comment (flavens) [inline README.md:48]: I would rather say: "and update the draft called xxx by changing: "

File: README.md [hunk 2]
```diff
@@ -45,22 +45,9 @@ We built the user guidelines and developer documentation with [MkDocs](https://w
...
+
+- the tag version, the release title (same name) with version number e.g. `v3.0.0` or `v3.0.0-rc.0`
```
Comment (flavens) [inline README.md:50]: I would rather say: "the tag version and the release title (same name) with the version number, e.g. ..."

**Review by flavens — CHANGES_REQUESTED** [review]
LGTM but I would change some sentences in the readme (see my comments)

File: README.md (hunk 1 above)
Comment (kilchenmann) [reply README.md:48]: Updated in 871740f

File: README.md (hunk 2 above)
Comment (kilchenmann) [reply README.md:50]: Updated in 871740f

**Review by flavens — APPROVED** [review]

**Pull request merged**